oeps/oep-0001.rst

```diff
@@ -226,68 +312,48 @@ Each OEP should have the following parts.
     a short descriptive title, the names, and optionally the contact info for each author.
 
 *Abstract*
-    A short (~200 word) description of the technical issue being addressed.
+    A short description of the technical issue being addressed.
```

Comment:
Same as in the template.  I think having a suggested ballpark for what short means is useful.